File: src/main/java/com/optica/OpticaApplication.java
package com.optica;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@SpringBootApplication
@EnableAutoConfiguration
public class OpticaApplication {

	/**
	 *
	 * el usuario se registra e importa contactos de su cel
     * enviara un mensaje de texto a n contactos
     * enviara un mensaje de texto personalizado basado  en columnas
     * enviara mensaje a travez de un web service
     * limita la cantidad de mensajes por membresia
     * informacion de membresia comprada
	 * el usuario se registra pero no se activara de inmediato solo enviara 3 msj
	 * el administrador habilita al usuario
	 *
	 *
     *
	 * */
	public static void main(String[] args) {
		SpringApplication.run(OpticaApplication.class, args);
	}


}

File: src/main/kotlin/com/optica/OpticaApplication.kt
package com.optica

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.EnableAutoConfiguration
import org.springframework.boot.autoconfigure.SpringBootApplication

@SpringBootApplication
@EnableAutoConfiguration
open class OpticaApplication {

    /**
     * TODO BACK-END
     * 1. load in session all user data
     * 2. People -> crud hv
     * 3. job-offer search by keyword and city
     * 4. job-offer new offer crud
     * 5. people apply job offer
     *
     * TODO FRONT-END
     * 1. contact form
     *
     */
    fun main(args: Array<String>) {
        SpringApplication.run(OpticaApplication::class.java, *args)
    }


}
